spec/Section 2 -- Language.md

```diff
@@ -716,28 +716,28 @@ sequences must be used to represent these characters.
 
 StringValue :: `""`
 
-  * Return an empty Unicode character sequence.
+  + Return an empty Unicode character sequence.
```

Comment:
`1.` looks very strange if it's only one element in the list:
![image](https://cloud.githubusercontent.com/assets/8336157/19870230/5e309618-9fb8-11e6-9664-89bfc1c80569.png)